### PR TITLE
Add auto declined settings to OUB

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: spec/components/schemas/GatewayAccountConfig/Onlineueberweisen.yaml [hunk 1]
```diff
@@ -36,5 +36,9 @@ allOf:
           delay:
             type: integer
             description: Gateway time delay to capture a transaction
+          autoDeclinedHour:
```
Comment (colinbird) [inline spec/components/schemas/GatewayAccountConfig/Onlineueberweisen.yaml]: autoDeclineTtl ?

File: spec/components/schemas/GatewayAccountConfig/Onlineueberweisen.yaml [hunk 2]
```diff
@@ -36,5 +36,9 @@ allOf:
           delay:
```
Comment (colinbird) [inline spec/components/schemas/GatewayAccountConfig/Onlineueberweisen.yaml]: Can we remove this one?

Comment (dara123) [reply spec/components/schemas/GatewayAccountConfig/Onlineueberweisen.yaml]: i'm not sure what this used for. let me check

Comment (colinbird) [reply spec/components/schemas/GatewayAccountConfig/Onlineueberweisen.yaml]: It was when we were using waiting-capture

Comment (dara123) [reply spec/components/schemas/GatewayAccountConfig/Onlineueberweisen.yaml]: ahh ok. I guess we can remove this

**Pull request merged**